README.md

```diff
@@ -107,16 +107,16 @@ refer to [CONTRIBUTING.md](CONTRIBUTING.md),
 or reach out to [@vijaytarian](https://twitter.com/vijaytarian)
 and [@ChenytangZhao](https://twitter.com/ChenytangZhao).
 
-
 ## Cite
 
 We have [written a paper describing Prompt2Model in detail](https://arxiv.org/abs/YYMM.XXXXX).
 
 If you use Prompt2Model in your research, please cite our paper:
-```
+
+```bibtex
 @misc{prompt2model,
-      title={Prompt2Model: Generating Deployable Models from Natural Language Instructions}, 
-      author={Vijay Viswanathan and Chenyang Zhao and Amanda Bertsch and Tongshuang Wu and Graham Neubig},
+      title={Prompt2Model: Generating Deployable Models from Natural Language Instructions}, {{< mdl-disable "<!-- markdownlint-disable MD013 -->" >}}
+      author={Vijay Viswanathan and Chenyang Zhao and Amanda Bertsch and Tongshuang Wu and Graham Neubig}, {{< mdl-disable "<!-- markdownlint-disable MD013 -->" >}}
```

Comment:
Let's add the following to `.markdownlint.yaml` instead of doing mdl-disable here:

```yaml
---
MD013:
  line_length: 120
  code_blocks: false
```